### ISSUE TITLE
Clear selected in multiselect when many items selected

### ISSUE THREAD

**Issue by Krangth** [brianvoe/slim-select#99]
There are about 30 options in my select, and about 7 more optgroup. I would like to be able to clear the selection. If someone has 12 selections, it can be tiresome to click on each small x one by one. The users do not want to reset the form as there are other inputs entered they wish to keep. Is there a way to clear the current selections?

In my efforts to devise a way, I was only able to accidentally select all the options rather than unselect all the options.

different topic...
closeOnSelect: false only works when not clicking on a optgroup heading. That's okay, but can an option be added to stay open on the optgroup click as well?

in conclusion, i really like slim select!

**Comment by brianvoe** [comment]
Thanks for the kind words

you can add your own clear button next to your select dropdown. and have that button call `select.set([])`

The close on select issues seems to be a bug as i probably didnt test using optgroups. If you could open a specific issue for that ill do some tests and make sure everything works properly with optgroups as well.
